bundles/binding/org.openhab.binding.mystromecopower/.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" output="target/classes" path="src/main/java"/>
+	<classpathentry excluding="**" kind="src" output="target/classes" path="src/main/resources"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="lib" path="D:/JavaProjects/openhab/bundles/binding/org.openhab.binding.mystromecopower/lib/gson-2.2.4.jar"/>
```

Comment:
This "lib" entry points to your local development environment. Please make it relative to the project directory like https://github.com/openhab/openhab/blob/master/bundles/binding/org.openhab.binding.withings/.classpath